api/python/provisioner/log.py

```diff
@@ -60,6 +61,30 @@ def filter(self, record):
         record.prvsnrcmd = self._cmd
         return True
 
+class LogFileFilter(logging.Filter):
```

Comment:
please don't combine two filters, python handlers [accept](https://docs.python.org/3.6/library/logging.config.html#dictionary-schema-details) list of filters